webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -0,0 +1,323 @@
+import { Tooltip } from 'antd';
```

Comment:
**question:** what do you think about using `useSettings` and `settings.ts` file to preserve pagination and sorting in the query params?

Reply:
done. it was very easy to use!